superset/cli.py

```diff
@@ -187,9 +187,18 @@ def import_dashboards(path, recursive):
 @click.option(
     '--print_stdout', '-p', is_flag=True, default=False,
     help='Print JSON to stdout')
-def export_dashboards(print_stdout, dashboard_file):
+@click.option(
```

Comment:
Could be nice to support multiple ids here:
https://click.palletsprojects.com/en/5.x/options/#multiple-options